client/app/services/usage-graphs/usage-graphs.html

```diff
@@ -0,0 +1,76 @@
+  <h2 translate>Utilization</h2>
+  <hr>
+  <div class="row">
+    <div class="col-md-4 text-center">
+                <div>
+                    <div class="metric-title" translate>CPU</div>
+                    <div class="metric-details-container">
+                        <div ng-if="vm.cpuDataExists">
+                            <span class="metric-number">{{vm.availableCPU}}</span>
+                            <span class="metric-details">
+                                <p translate>MHz Available</p>
+                                <p><span translate>of</span> {{vm.cpuChart.data.total}} <span translate>MHz</span></p>
+                            </span>
+                        </div>
+                    </div>
+                    <hr>
+                    <div ng-if="!vm.cpuDataExists">
+                      <div>
+                          <i class="pf pficon-info"></i>
+                      </div>
+                      <div translate>
+                        No data available
+                      </div>
+                    </div>
+                </div>
+                <div pf-donut-pct-chart ng-if="vm.cpuDataExists" config="vm.cpuChart.config" data="vm.cpuChart.data" center-label="vm.cpuChart.label"></div>
+    </div>
+    <div class="col-md-4 text-center">
+        <div>
+            <div class="metric-title" translate>Memory</div>
+            <div class="metric-details-container">
+                <div ng-if="vm.memoryDataExists">
+                    <span class="metric-number">{{vm.availableMemory}}</span>
+                    <span class="metric-details">
+                        <p translate>GB Available</p>
+                        <p><span translate>of</span> {{vm.memoryChart.data.total}} <span translate>GB</span></p>
+                    </span>
+                </div>
+            </div>
+            <hr>
+            <div ng-if="!vm.memoryDataExists">
+                <div>
+                    <i class="pf pficon-info"></i>
+                </div>
+                <div translate>
+                No data available
```

Comment:
Maybe not for this pr, but something we might want to look into doing is create a no data available component, i gotta feeling that might have uses in more places than here...